### PR TITLE
48 starting service

### PR DESCRIPTION
**Pull request by Blynqs**
Added Service Starter, a executable that starts the hub process in 2 clicks.
Added Installer script.

### CODE REVIEW

File: ApiGateway/appsettings.json
Comment (GuyomT) [inline ApiGateway/appsettings.json:1]: Is that still ok with docker for dev purposes ?

File: Auth/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    }
+  },
+  "Kestrel": {
+    "Endpoints": {
+      "Http": {
+        "Url": "http://localhost:7002"
+      }
+    }
+  },
+  "AgentURL": "https://httpbin.org/",
```
Comment (GuyomT) [inline Auth/appsettings.json:15]: this can be removed

File: DataProcessing/appsettings.json
Comment (GuyomT) [inline DataProcessing/appsettings.json:1]: same question ? docker ok ?

**Review by GuyomT — COMMENTED** [review]
please answer to my question before merging !
code lgtm 👍

File: DataProcessing/appsettings.json
Comment (Blynqs) [reply DataProcessing/appsettings.json:1]: yes, docker will use the appsettings.Container, as it was already doing with Auth Service

File: Auth/appsettings.json (hunk 1 above)
Comment (Blynqs) [reply Auth/appsettings.json]: done

File: ApiGateway/appsettings.json
Comment (Blynqs) [reply ApiGateway/appsettings.json:1]: yes, see my answer below

**Pull request merged**